test/parse_function.cpp

```diff
@@ -6,42 +6,48 @@
 #include "../src/parse.hpp"
 #include "../src/token.hpp"
 
-TEST(Function, DISABLED_Function)
+TEST(Function, Function)
 {
     std::vector<Token>
         tok_vec{
             Token("KEYWORD", "FUNCTION"),
             Token("IDENTIFIER", "F_Sample"),
-            Token("SPECIAL_CHARACTER", ":"),
-            Token("KEYWORD", "INT"),
-            Token("KEYWORD", "VAR_INPUT"),
-            Token("IDENTIFIER", "x"),
-            Token("SPECIAL_CHARACTER", ":"),
-            Token("KEYWORD", "INT"),
-            Token("SPECIAL_CHARACTER", ";"),
-            Token("IDENTIFIER", "y"),
-            Token("SPECIAL_CHARACTER", ":"),
-            Token("KEYWORD", "INT"),
-            Token("SPECIAL_CHARACTER", ";"),
-            Token("KEYWORD", "END_VAR"),
-            Token("IDENTIFIER", "F_Sample"),
-            Token("OPERATOR", ":="),
-            Token("IDENTIFIER", "x"),
-            Token("OPERATOR", "+"),
-            Token("IDENTIFIER", "y"),
-            Token("SPECIAL_CHARACTER", ";"),
+            // Token("SPECIAL_CHARACTER", ":"),
```

Comment:
Don't forget to uncomment these before merging.

Reply:
This has been corrected will be committed shortly